scripts/utils/createJestConfig.js

```diff
@@ -21,8 +21,8 @@ module.exports = (resolve, rootDir) => {
   const config = {
     moduleFileExtensions: ['jsx', 'js', 'json'],
     moduleNameMapper: {
-      '^[./a-zA-Z0-9$_-]+\\.(jpg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm)$': resolve('config/jest/FileStub.js'),
-      '^[./a-zA-Z0-9$_-]+\\.css$': resolve('config/jest/CSSStub.js')
+      '^.+\\.(jpg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm)$': resolve('config/jest/FileStub.js'),
```

Comment:
That's a good idea. However, turns out the files don't get stubbed, if I change this to `\\.css$`. This looks like a possible bug in Jest to me (unless it's only supposed to work with regexes that match the whole path). When given a non-anchored regex, [this line doesn't replace](https://github.com/facebook/jest/blob/06033a155d049e83d995b8775cc7e6b0547d7d7b/packages/jest-resolve/src/index.js#L206) the  module name correctly.

I suggest that we merge this PR as-is, with the anchored regex, and we can open an issue in Jest about regexes like `\\.css$` not working correctly.